tests/autobahn/test_autobahn.py

```diff
@@ -0,0 +1,38 @@
+import shutil
+from pathlib import Path
+
+import pytest
+from python_on_whales import DockerClient
+
+
+@pytest.fixture(scope="session", autouse=True)
+def create_report_directory():
+    path = Path("tests/autobahn/reports")
+    if path.is_dir():
+        shutil.rmtree(path)
+    path.mkdir()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def build_aiohttp_docker_image():
+    docker = DockerClient()
+    docker.buildx.build(
+        context_path=".", file="tests/autobahn/Dockerfile.aiohttp", tags=["aiohttp"]
+    )
+    try:
+        yield
+    finally:
+        # default is the name of the default builder
+        docker.buildx.remove("default")
+
+
+def test_client():
+    docker = DockerClient(compose_files=["tests/autobahn/client/docker-compose.yml"])
+    docker.compose.up(abort_on_container_exit=True)
```

Comment:
I wonder if you could inject some assertion checking for the presence of the logs, for example, or maybe even analyzing their contents for errors.

Reply:
I think we could extract some information from the generated reports and make assertions on that.